README.md

```diff
@@ -39,6 +41,11 @@ The Falcon Helm chart has been tested to deploy on the following Kubernetes dist
   * Nodes must be Linux distributions supported by CrowdStrike. See [https://falcon.crowdstrike.com/support/documentation/20/falcon-sensor-for-linux#operating-systems](https://falcon.crowdstrike.com/support/documentation/20/falcon-sensor-for-linux#operating-systems) for supported Linux distributions and kernels.
 * Red Hat OpenShift Container Platform 4.6+
 
+The Falcon Helm Operator has been tested to deploy on the following Kubernetes
+distributions:
+
+* Red Hat OpenShift Container Platform 4.6+
```

Comment:
Does it make sense to call out the full version number, e.g. ``4.6.20+``? Not sure how backward compatible things are.